internal/config/parse.go

```diff
@@ -114,8 +114,31 @@ func loadPlugins(ctx context.Context, raw *rawConfig) error {
 
 	if len(raw.MachComposer.Plugins) == 0 {
 		log.Debug().Msg("No plugins specified; loading default plugins")
-		if err := raw.plugins.LoadDefault(ctx); err != nil {
-			return err
+		raw.MachComposer.Plugins = map[string]MachPluginConfig{
```

Comment:
Do we want to maintain this list? Maybe it is better to just throw an error if 0 plugins are installed? Now the version numbers are also broken, I think